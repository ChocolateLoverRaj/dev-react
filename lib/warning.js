// Get a custom task for unrecognized file warnings
import CustomTask from './tasks/tasks/custom.js'
import TextAddon from './tasks/addons/text.js'
import ColorAddon from './tasks/addons/color.js'

const warning = (warning, filename) => {
  return new CustomTask(new ColorAddon(new TextAddon(`\u26a0 ${warning}: ${filename}`), 'yellow'))
}

export default warning
