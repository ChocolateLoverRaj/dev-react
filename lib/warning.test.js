/* eslint-env jest */

import warning from './warning.js'

test('colorful task', () => {
  expect(warning('my warning', 'a file').getLines()).toMatchSnapshot()
})
