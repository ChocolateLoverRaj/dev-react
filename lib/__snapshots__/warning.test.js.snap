// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colorful task 1`] = `
Array [
  "[93mâš  my warning: a file[39m",
]
`;
